node/resolvers/session/index.ts

```diff
@@ -98,4 +116,25 @@ export const mutations = {
 
     return true
   },
+
+  savePickupInSession: async (_: any, args: SavePickupArgs, ctx: Context) => {
+    const { address, name } = args
+    const {
+      clients: { session },
+      cookies,
+    } = ctx
+
+    await session.updateSession(
+      'favoritePickup',
+      { address: convertCheckoutAddressToProfile(address), name },
+      [],
+      cookies.get(VTEX_SESSION)!
+    )
+
+    return queries.getSession({}, {}, ctx)
```

Comment:
what do these arguments mean? (I realize this is not related to this PR, but perhaps add a lil comment of what each argument is?)

:pick:

Reply:
its just calling the getSession method above, the function expects _ and __ in first two args